tests/Microsoft.Identity.Test.Unit/pop/PopAuthenticationOperationTests.cs

```diff
@@ -75,7 +75,7 @@ public void ValidatePopRequestAndToken()
 
                 // Assert
                 Assert.AreEqual("PoP", authenticationScheme.AuthorizationHeaderPrefix);
-                Assert.AreEqual(TokenType.Pop, (TokenType)authenticationScheme.TelemetryTokenType);
+                Assert.AreEqual(2, authenticationScheme.TelemetryTokenType);
```

Comment:
```suggestion
                Assert.AreEqual(TelemetryTokenTypeConstants.pop, authenticationScheme.TelemetryTokenType);
```

Comment applies to all references in test